environment/environment_templates.go

```diff
@@ -315,7 +316,7 @@ func NewChainlinkClusterForAlertsTesting(nodeCount int) K8sEnvSpecInit {
 	}
 
 	dependencyGroup := getBasicDependencyGroup()
-	addServicesForTestingAlertsToDependencyGroup(dependencyGroup, nodeCount)
+	addServicesForTestingObservabilityToDependencyGroup(dependencyGroup, nodeCount)
```

Comment:
Do we really need a method with such a long name that only pushes to a slice? Maybe slice is enough I think

Reply:
removed this function